membership/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.conf import settings
+# Create your models here.
+
+MEMBERSHIP_CHOICES = (
+('Premium', 'pre'),
+('Free', 'free')
+)
+class Membership(models.Model):
+    slug = models.SlugField(null=True, blank=True)
+    membership_type = models.CharField(
+    choices=MEMBERSHIP_CHOICES, default='Free',
+    max_length=30
+      )
+    price = models.DecimalField(max_digits=6, decimal_places=2, blank=True, null=True)
+def __str__(self):
+       return self.membership_type
```

Comment:
Second time we've gone through this one. 

`def __str__(self): ... ` is a method on a model. You _need_ to indent properly for this to work otherwise your admin will say `Membership (object 1)`